easybuild/tools/module_generator.py

```diff
@@ -1110,6 +1130,12 @@ def get_description(self, conflict=True):
         for line in self._generate_whatis_lines():
             whatis_lines.append("whatis(%s%s%s)" % (self.START_STR, self.check_str(line), self.END_STR))
 
+        provide_list = self._generate_provides_list()
+
+        if provide_list:
+            provide_list_mod = 'extensions(%s)' % ', '.join(['"%s"' % x for x in provide_list])
+            lines.extend(['', self.conditional_statement(self.check_version("8", "2"), provide_list_mod)])
```

Comment:
add a comment explaining why we're wrapping this under an Lmod version check (bonus points if you include a reference to the Lmod docs)